chandra_aca/aca_image.py

```diff
@@ -828,3 +832,82 @@ def get_psf_image(
 
         out = ACAImage(psf, row0=row0, col0=col0) if aca_image else (psf, row0, col0)
         return out
+
+
+@retry.retry(exceptions=requests.exceptions.RequestException, delay=5, tries=3)
+def get_aca_image_table(
+    start: CxoTimeLike, stop: CxoTimeLike, bgsub=True, source="maude", **maude_kwargs
+):
+    """
+    Get ACA images and ancillary data from either the mica or maude data sources.
+
+    Parameters
+    ----------
+    start : CxoTimeLike
+        start time
+    stop : CxoTimeLike
+        stop time (CXC sec)
+    bgsub : bool
+        include background subtracted images in output table
+    source : str
+        'maude' or 'mica'
```

Comment:
```suggestion
        Data source for image telemetry ('maude' or 'mica')
```

Reply:
It is also the source for the temperature telemetry but yes.

Reply:
Actually temperatures are coming from either maude or cheta fetch CXC (not mica). Maybe we don't care, but being pedantic it might work to make the options here 'maude' or 'cxc'. I updated the suggestion to show what this would look like.